flow/connectors/clickhouse/qrep.go

```diff
@@ -68,31 +68,23 @@ func (c *ClickhouseConnector) createMetadataInsertStatement(
 	return insertMetadataStmt, nil
 }
 
-func (c *ClickhouseConnector) getTableSchema(tableName string) ([]*sql.ColumnType, error) {
-	//nolint:gosec
+func (c *ClickhouseConnector) getTableSchema(ctx context.Context, tableName string) ([]driver.ColumnType, error) {
 	queryString := fmt.Sprintf(`SELECT * FROM %s LIMIT 0`, tableName)
-	//nolint:rowserrcheck
-	rows, err := c.database.Query(queryString)
+	rows, err := c.database.Query(ctx, queryString)
 	if err != nil {
 		return nil, fmt.Errorf("failed to execute query: %w", err)
 	}
 	defer rows.Close()
 
-	columnTypes, err := rows.ColumnTypes()
-	if err != nil {
-		return nil, fmt.Errorf("failed to get column types: %w", err)
-	}
-
-	return columnTypes, nil
+	return rows.ColumnTypes(), nil
```

Comment:
is the output identical across the two functions

Reply:
Different types, but same for our purposes